.golangci.yml

```diff
@@ -4,10 +4,10 @@ output:
 linters:
   enable:
     - goimports
-    - golint
```

Comment:
`golint` is deprecated and superseded with `revive`. Lots of linter warnings with revive, which have been ignored. Issue to fix this is tracked [here](https://github.com/grafana/alerting-squad/issues/724).